components/public-api/gitpod/experimental/v1/teams.proto

```diff
@@ -142,6 +157,16 @@ message ResetTeamInvitationResponse {
     TeamInvitation team_invitation = 1;
 }
 
+message ListTeamMembersRequest {
+    // team_id is the ID of the team that contains the members to list
+    string team_id = 1;
+}
+
+message ListTeamMembersResponse {
+    // members are the team members of this Team
+    repeated TeamMember members = 1;
```

Comment:
👍 let's tackle [this](https://linear.app/gitpod/issue/EXP-778/members-list-should-be-paginated) in a follow up.

Reply:
Not sure if pagination is worth doing for API, that our search on dasbboard will search for user name and email both, then we need to do `debounce` on dashboard and sql `<fields> like %?%` on server

Reply:
In dedicated we can easily have hundreds of organization members